lib/core/components/app_bar/app_bar_action_button.dart

```diff
@@ -19,7 +18,7 @@ class AppBarActionButton extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
     return Padding(
-      padding: const EdgeInsets.symmetric(vertical: 7.0, horizontal: 5.0),
+      padding: const EdgeInsets.symmetric(vertical: 7.0, horizontal: 7.0),
```

Comment:
If they are the same, let's use 
```suggestion
      padding: const EdgeInsets.all(7.0),
```
It can also be 8.0.